plugins/experiment_procedure.cc

```diff
@@ -84,6 +84,15 @@ ExperimentProcedureConfigDescription()
              "Model mode: `boolean`, `regression` or `categorical`. "
              "Controls how the label is interpreted and what is the output of the classifier. "
              , CM_BOOLEAN);
+    addField("multilabelStrategy", &ExperimentProcedureConfig::multilabelStrategy,
```

Comment:
No.  We can embed an enum's documentation in the containing page.  I don't understand why we need this option or what it does.  Maybe a higher level description, or could we just not have an option at all and leave it as a default?  What is the user story for this?